index.html

```diff
@@ -50,6 +50,212 @@ <h3 class="box3">
         </ul>
     </section>
 
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot2.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Tonic
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot1.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Tonic
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
```

Comment:
According to the requirement, you should have a section for the projects in that section have different division for each projects as shown below instead of having each project in different `section` tag:
For example: 
`<section class="projects">`
         `then you can have a div for project 1 like this:`
        ` <div class="project-one">`
               `All the details of the project one goes here`
        `</div>`
          `<div class="project-two">`
               `All the details of the project two goes here`
        `</div>`
`</section>`

Then the content of each project is aligned with flexbox.